api/src/main/java/org/openmrs/Relationship.java

```diff
@@ -9,27 +9,52 @@
  */
 package org.openmrs;
 
+import javax.persistence.AttributeOverride;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
 import java.util.Date;
 
 /**
  * Relationship
  */
+@Entity
+@Table(name = "relationship")
+@AttributeOverride(name = "voidReason", column = @Column(name = "void_reason", length = 255))
 public class Relationship extends BaseChangeableOpenmrsData {
-	
+
 	public static final long serialVersionUID = 323423L;
-	
+
 	// Fields
-	
+
```

Comment:
looks good though with some code formats,you could use mvn clean  package before pushing to git